### PR TITLE
GolfDiva-Initial add

### PR DESCRIPTION
**Pull request by GolfDiva**
I think I am catching up... Let's cross our fingers. You can comment if you like!

### CODE REVIEW

**Comment by yashkmankad** [comment]
Glad you were able to catch up!

File: GolfDiva-LaHabraInfo [hunk 1]
```diff
@@ -0,0 +1,8 @@
+** La Habra
```
Comment (githubteacher) [inline GolfDiva-LaHabraInfo]: @GolfDiva where is La Habra? I should really learn more about the 🌎 😢

**Comment by sdcalmes** [comment]
What are some local attractions in La Habra? :city_sunrise:

File: GolfDiva-LaHabraInfo (hunk 1 above)
Comment (KimThomp) [inline GolfDiva-LaHabraInfo]: @GolfDiva is that you Geri?

Comment (GolfDiva) [inline GolfDiva-LaHabraInfo]: **Kimmy!** Yes, Ma'am! It is! Nice to see a friend!

Comment (GolfDiva) [inline GolfDiva-LaHabraInfo]: La Habra is 10 Miles north of Disneyland. It is the north/west most corner of Orange County

Comment (mannu9898) [inline GolfDiva-LaHabraInfo]: Maybe this ?

**Pull request merged**